From efcefb0d20ec83f88ab24d796c7596400efdb3c6 Mon Sep 17 00:00:00 2001
From: Bastien Nocera <hadess@hadess.net>
Date: Wed, 19 Jul 2017 18:38:42 +0200
Subject: [PATCH] djvu: Remove image/vnd.djvu mime-type handling

Now that shared-mime-info has included an image/vnd.djvu+multipage
mime-type for more than a year.

https://bugzilla.gnome.org/show_bug.cgi?id=754467
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index b34fcbc7..a480cbf1 100644
--- a/configure.ac
+++ b/configure.ac
@@ -780,7 +780,7 @@ AC_SUBST(DVI_MIME_TYPES)
 AC_SUBST(APPDATA_DVI_MIME_TYPES)
 AM_SUBST_NOTMAKE(APPDATA_DVI_MIME_TYPES)
 if test "x$enable_djvu" = "xyes"; then
-        DJVU_MIME_TYPES="image/vnd.djvu;image/vnd.djvu+multipage;application/x-ext-djv;application/x-ext-djvu"
+        DJVU_MIME_TYPES="image/vnd.djvu+multipage;application/x-ext-djv;application/x-ext-djvu"
         APPDATA_DJVU_MIME_TYPES=$(echo "<mimetype>$DJVU_MIME_TYPES</mimetype>" | sed -e 's/;/<\/mimetype>\n    <mimetype>/g')
         if test -z "$EVINCE_MIME_TYPES"; then
            EVINCE_MIME_TYPES="${DJVU_MIME_TYPES}"
-- 
2.13.0
